client/src/main/java/org/schemarepo/client/Avro1124RESTRepositoryClient.java

```diff
@@ -93,7 +93,7 @@ public Subject lookup(String subject) {
   public Iterable<Subject> subjects() {
     ArrayList<Subject> subjectList = new ArrayList<Subject>();
     try {
-      String subjects = webResource.get(String.class);
+      String subjects = webResource.accept(MediaType.TEXT_PLAIN_TYPE).get(String.class);
```

Comment:
Ah... that's a bummer.

Although it hasn't been stated explicitly (yet), I'd like one of the design goals of this project to be easy upgrade-ability for users of the AVRO-1124 code. That being said, I think we should balance that need against the goal of offering as much as possible (ideally all) of the new features while still maintaining that compatibility. Your proposed solution of having a parameter to enable HTML support works for the backwards compatibility goal, but it's too bad that it would essentially make HTML browsing unavailable for anyone who hasn't fully migrated ALL of their clients to the new implementation... don't you think?

I'm not a hardcore RESTafarian so I don't know what the purists would suggest. I imagine they would say it's best to have a single endpoint with multiple supported content types. That being said, being pragmatic, I think the best thing to do in this case would be to have several endpoints...

So perhaps we can have a root path which only supports plain-text (and thus, older clients), and another root path which supports all content types? Ideally, we should aim to support both with the same code base, by simply instantiating multiple ```RESTRepository``` instances with the same ```Repository``` instance passed in but different sets of ```Renderer```s. Or something along those lines (I'm not attached to any specific implementation strategy, but the goal would be to maximizes code re-use and cache things only once).

If making both work at the same time seems like too much work, we can begin with your proposition of making HTML support globally toggled (with the "on" setting breaking old clients), but if we do that, I think the default config should be "off" (unfortunately...), and that we should follow up with a separate issue to enable both simultaneously/gracefully.

What do you think?